crates/noirc_frontend/src/hir/def_collector/dc_crate.rs

```diff
@@ -240,7 +240,7 @@ fn collect_impls(
 
 fn resolve_global_constants(
```

Comment:
Kind of a nitpick. Do we want to rename here too especially if we change globals to later not just be constants? We also still have `push_global_const`, `update_global_const`, etc in the interner. As globals can only be literals right now it isn't too confusing, but it will be if we enable global exprs and func calls.

Reply:
Should be resolved now